utilities/monitoring.py

```
@@ -177,6 +181,32 @@ def get_metrics_value(prometheus, metrics_name):
+        wait_timeout=timeout,
+        sleep=TIMEOUT_15SEC,
+        func=get_metrics_value,
+        prometheus=prometheus,
+        metrics_name=metric_name,
+    )
+    sample = None
+    comparison_values_log = {}
+    try:
+        for sample in samples:
+            if sample:
+                comparison_values_log[datetime.datetime.now()] = (
```

Ruff (0.15.20)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)